### DISCUSSION TITLE
Module design in skywalking-rover's docs is broken

### DISCUSSION THREAD

**Discussion by flearc** [apache/skywalking#12781] [Q&A] [answered]
The `module design` link is broken.

I can find the file from https://github.com/apache/skywalking-rover/pull/119. It seems the file was deleted because "It is a dev concept, not a user's interest", 
![image](https://github.com/user-attachments/assets/455c4485-3b41-4ab1-b6cc-32cb51158ff2)
While it may not be of interest to users, contributors need it to understand the rover repo. Please keep it (and update it if necessary)

**Comment by wu-sheng** [comment]
AFAIK, we don't have this anymore.

Please follow the website docs link.

**Reply by flearc** [reply]
Then please fix the broken link. It is confused I get a not exist page.
![image](https://github.com/user-attachments/assets/28eef603-cdf1-48a8-988c-52a23a3504ed)

![image](https://github.com/user-attachments/assets/948ee46a-064e-4a52-a775-a338c3f0e054)

**Reply by wu-sheng** [reply]
Where do you find this link?

**Reply by flearc** [reply]
rover contribution docs: https://skywalking.apache.org/docs/skywalking-rover/next/en/guides/contribution/how-to-write-module/

**Reply by wu-sheng** [reply]
OK. Got it. Let's see how to fix it.

**Reply by wu-sheng** [reply]
Fixed by https://github.com/apache/skywalking-rover/pull/158

The next version will be updated at 1800 Beijing time.

**Comment by mrproliu** [comment, marked as answer]
We have fixed in the PR: https://github.com/apache/skywalking-rover/pull/158
